react/components/Footer/FooterUI.jsx

```diff
@@ -28,11 +20,11 @@ const FooterUI = (props) => {
                     src="/images/hhs-logo-black.svg"
                 />
             ),
-            qppLogo: (
+            madieLogo: (
                 <img
                     className="qpp-logo"
                     src="/images/qpp_logo_rgb_color.png"
-                    alt="qpp logo"
+                    alt="madie logo"
```

Comment:
we can probably just remove this img, not related to madie?

Reply:
This is actually the madie logo